### ISSUE TITLE
RSS Error

### ISSUE THREAD

**Issue by sjehuda** [BlackLight/madblog#1]
```
This page contains the following errors:
error on line 44 at column 78: Input is not proper UTF-8, indicate encoding !
Bytes: 0x02 0x34 0x30 0x03
Below is a rendering of the page up to the first error.
```
[Screenshot](https://github.com/BlackLight/madblog/files/11246474/platypush.pdf) of https://blog.platypush.tech/rss using [Newspaper](https://openuserjs.org/scripts/sjehuda/Newspaper) userscript.

**Comment by blacklight** [comment]
Yeah in that specific case it comes from some LaTeX formula in one of the articles that breaks the XML. Since it's a very very rare event I'm not sure whether I should find a way to escape that specific formula, or just remove it - unless you managed to replicate the same error with another madblog installation too.

**Comment by blacklight** [comment]
I have released madblog 0.2.17 that fixes the issue. The solution has been to remove the `alt` attribute from the LaTeX rendered images, since that can generate non-standard Unicode characters that break the XML encoding.
